draft-packages/table/docs/Table.stories.tsx

```diff
@@ -463,10 +467,7 @@ export const Tooltip: StoryFn = () => (
             wrapping="wrap"
             tooltipInfo="This is a tooltip"
             sorting="ascending"
-            onClick={(e: React.MouseEvent): void => {
-              e.preventDefault()
```

Comment:
Why are you not preventing default anymore?

Reply:
This was a mistake by me in the previous update, I copied the code from the anchor click example. I've now used the sorting click example code instead.